.gitignore

```diff
@@ -92,3 +94,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+#Django Migrations
+0001_initial.py
```

Comment:
Why did you add migrations to gitignore?

Reply:
@r-wambui the PR couldn't be merged automatically with that file in...plus it's autogenerated by django

Reply:
But we do not ignore migrations. You could have looked for a way to solve the conflicts.

Reply:
Ok let me find a way around this. :)